src/main/scala/org/apache/mesos/chronos/scheduler/state/MesosStatePersistenceStore.scala

```diff
@@ -59,11 +64,26 @@ class MesosStatePersistenceStore @Inject()(val zk: CuratorFramework,
     }
   }
 
+  def removeTaskFlowState(id: String): Boolean = {
+    remove(flowName(id))
+  }
+  
+  def getTaskFlowState(id: String): TaskFlowState = {
```

Comment:
I'm not fully convinced that adding task flow parameters to Chronos is the way to go here. Perhaps in your job configuration, you could create the environment variables for your job and make sure the proper environment variables in your chain of jobs are intact when you define your jobs. This could be done through scripting and you could utilize zookeeper or another key value store for this. The issue with adding this patch to Chronos is it adds another layer of dependency to zookeeper and also more complexity to setting the job arguments. One alternative thought is that we could add an inherit environment flag so child jobs can inherit their parents environment if the flag is set. Let's discuss!